byron/ledger/executable-spec/test/Ledger/Pvbump/Properties.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Ledger.Pvbump.Properties where
+
+import           Control.State.Transition (applySTS, TRC(..))
+import           Generator
+import           Hedgehog
+import           Ledger.Update (PVBUMP)
+
+
+-- Property #1 for the PVBUMP STS
+-- 
+-- Given an empty list of mappings from a slot to a pair of a protocol
+-- version and protocol parameters (this is the fads part of the
+-- environment), the PVBUMP STS results in the same state it started
+-- in.
+emptyPVUpdate :: Property
+emptyPVUpdate = property $ do
+  PVBUMPEnv env  <- forAll pvbumpEmptyListEnvGen
+  PVBUMPState st <- forAll pvbumpStateGen
+  sig            <- forAll pvbumpSignalGen
+  case applySTS @PVBUMP $ TRC (env, st, sig) of
+    Right st' -> st === st'
+    Left _    -> failure
+
+-- Property #2 for the PVBUMP STS
+--
+-- For s_n <= 2 * k, the resulting state is the same as the state the
+-- system started in.
+beginningsNoUpdate :: Property
+beginningsNoUpdate = property $ do
+  PVBUMPEnv env  <- forAll pvbumpBeginningsEnvGen
```

Comment:
Similar here to avoid shrinking dependencies

Reply:
@ruhatch , please help here, the issues is the same as in https://github.com/input-output-hk/cardano-ledger-specs/pull/488/files#r286938596

Reply:
Fixed!